web/admin.py

```diff
@@ -213,6 +213,6 @@ class EndpointAdmin(admin.ModelAdmin):
 
 @admin.register(Exposure)
 class ExposureAdmin(admin.ModelAdmin):
-    list_display = ('dataset', 'id', 'endpoint', 'form_id')
-    search_fields = ('id', 'dataset', 'endpoint', 'form_id')
+    list_display = ('dataset', 'id', 'endpoint', 'form_id', 'form_name')
+    search_fields = ('id', 'dataset', 'endpoint', 'form_id', 'form_name')
     ordering = ('added',)
```

Comment:
`list_filter = ['endpoint__name']` can be added here so that datastewards can get all the exposures related to one particular endpoint.